scripts/alert_emails_report/generate-subscriber-emails-file.ts

```diff
@@ -0,0 +1,21 @@
+import _ from 'lodash';
+import { fetchAllAlertSubscriptions } from '../alert_emails/firestore';
+import { getFirestore } from '../common/firebase';
+
+async function main() {
+  const db = getFirestore();
+  const subscriptions = await fetchAllAlertSubscriptions(db);
+  const fs = require('fs');
+  let output = '';
+  subscriptions.map(subscription => {
```

Comment:
You are using `map()` but you aren't actually returning a value from the callback function or using the return value of `map()`, so I think this could just be `forEach()`.

Alternatively you could probably write this as something like `const output = subscriptions.map(s => s.email).join(',');`

Reply:
😅